app/assets/javascripts/models/beta/story.js

```diff
@@ -108,6 +108,16 @@ export const findById = (stories, id) => {
   return stories.find(story => story.id === id)
 }
 
+export const sort = async (story) => {
```

Comment:
It's kinda confusing to call `sort` a function that operates on a single item? Maybe it would be clearer to call it `updatePosition`, what do you think?

Reply:
The name `sort` is given because in addition to updating the story passed in the parameter, it also updates all those that have a position greater than this in the same column

Reply:
I see the position change for the other stories more like a side effect than the feature itself. For example, it's not exactly a sorting (in the sorting algorithm sense), it's more a matter of changing a single story for a new position and the consequence of that is increasing the position value of the other ones. The word "sort" implies you're using some characteristic of all the involved items to put them in a specific order but, in this case, we're sending a single story to have a new position assigned to it. Does it make sense?

Reply:
Ok. I will change it